rs/artifact_pool/src/ingress_pool.rs

```diff
@@ -41,16 +45,32 @@ struct IngressPoolSection<T: AsRef<IngressPoolObject>> {
     /// and purge invocations. Never modify the artifacts map directly! Use the
     /// associated functions [`insert`], [`remove`] and [`purge_below`]
     byte_size: usize,
```

Comment:
Is this still used?

Reply:
Good point.
For some reason I thought it was used to report metrics, but it's not.

I've removed the field